src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -818,12 +818,9 @@ public function parseTable($table)
     {
         [$connection, $table] = Str::contains($table, '.') ? explode('.', $table, 2) : [null, $table];
 
-        if (Str::contains($table, '\\') && class_exists($table)) {
+        if (Str::contains($table, '\\') && class_exists($table) && is_a($table, Model::class, true)) {
```

Comment:
`$table instanceof Model` should be used here

Reply:
I don't think you can use instanceof on strings

Reply:
Oh, right, yeh. ;)